hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh

```diff
@@ -66,6 +66,19 @@ get_es_indices() {
     | xargs -I '{}' oc -n openshift-operators-redhat exec '{}' -c elasticsearch -- es_util --query=_cat/indices
 }
 
+insert_record_to() {
+  index=$1
+  oc -n openshift-operators-redhat get pods -l component=elasticsearch --no-headers=true --ignore-not-found \
+    | awk 'NR==1{print $1}' \
+    | xargs -I '{}' oc -n openshift-operators-redhat exec '{}' -c elasticsearch -- es_util --query=$index/_doc/1 -d '{"key":"value"}' -XPUT
```

Comment:
Should be `$index/doc/1`
```suggestion
    | xargs -I '{}' oc -n openshift-operators-redhat exec '{}' -c elasticsearch -- es_util --query=$index/doc/1 -d '{"key":"value"}' -XPUT
```